src/upser-defaults/upser-defaults.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+import { InjectConnection } from '@nestjs/mongoose';
+import { Connection } from 'mongoose';
+
+@Injectable()
+export class UpserDefaultsService implements OnModuleInit {
+  constructor(@InjectConnection() private readonly connection: Connection) {}
+
+  async onModuleInit(): Promise<void> {
+    await this.upsertDefaults();
+  }
+
+  async upsertDefaults(): Promise<void> {
+    const dbName = 'dev';
+    const db = this.connection.useDb(dbName, { useCache: true });
```

Comment:
dodatkowo musisz sprawdzić czy istnieje wgl baza i jak nie to ja przecież stworzyć

Reply:
w mongoose jeżeli użyjesz useDb to jeżeli takiej db nie ma to automatycznie ją tworzy